### DISCUSSION TITLE
anyway to get the _all_docs?include_docs=true   file?

### DISCUSSION THREAD

**Discussion by pingpongchampion1995** [community/community#128425] [npm]
i used to be able to get it (although it took forever) through the link and downloaded it, but nowadays it moves at a snails pace and most of the time ends up quitting. is this file uploaded somewhere online semi-regularly?

**Comment by donmccurdy** [comment]
+1, I'm looking for the same thing. I would like to download an archive of `package.json` data to do some analysis on popularity various open source licenses (an old archive would be fine), but it no longer seems possible to download this file.
